components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -90,8 +93,9 @@ @implementation BannerTypicalUseExampleViewController
 - (instancetype)init {
   self = [super init];
   if (self) {
-    self.colorScheme = [[MDCSemanticColorScheme alloc] init];
-    self.typographyScheme = [[MDCTypographyScheme alloc] init];
+    self.containerScheme = [[MDCContainerScheme alloc] init];
```

Comment:
I believe within `init` we should be using the `ivar`.

Reply:
Addressed in 5361ac676f23c5a5b56c1e374032d65f049a8975.